### PR TITLE
Update wording on OIDC logout

### PR DESCRIPTION
**Pull request by zachmargolis**

### CODE REVIEW

File: _pages/oidc.md [hunk 1]
```diff
@@ -353,7 +353,7 @@ This public key is rotated periodically (on at least an annual basis), so be sur
 
 login.gov supports [RP-Initiated Logout](https://openid.net/specs/openid-connect-session-1_0.html#RPLogout), allowing clients to log users out of their current login.gov session and redirect them back to the Relying Party.
 
-login.gov does not support Single Logout for OpenID connect. Taking this action will not end any sessions on any other applications that a user may have already started with their same login.gov account. For example, if a user signs in to applications A and B through login.gov, a logout request from A will end their login.gov session, but will not affect the session in application B.
+login.gov does not support Single Logout for OpenID Connect. Logging out will not end any other sessions that a user may have open on another application. For example, if a user signs in to applications A and B through login.gov, a logout request from A will end their login.gov session, but will not affect the session in application B.
```
Comment (amoose) [inline _pages/oidc.md:356]: Login.gov should be capitalized at the beginning of sentences. 

> Login.gov does not support Single Logout (SLO). The logout action will terminate the user's session at login.gov but will not end any other potentially active sessions within service provider applications.

Comment (zachmargolis) [reply _pages/oidc.md:356]: I thought that SAML did support SLO....? But if SAML does not support, I'll update the SAML doc too.

Comment (zachmargolis) [reply _pages/oidc.md]: I like your change suggestion, added in 3e293d54ca041958076b279553760a4bea2f476d

Comment (TiffanyAndrews) [reply _pages/oidc.md]: LGTM

**Pull request merged**